ui/StatusQ/src/StatusQ/Controls/StatusBaseButton.qml

```diff
@@ -36,6 +36,7 @@ Button {
     property alias tooltip: tooltip
 
     property bool loading
+    property bool loadingWithText // loading indicator instead of icon, mutually exclusive with `loading`
```

Comment:
I flags are mutually exclusive, it would be nice to have an enum and single property for that but I presume it's used in too many places to do the refactor,